src/js/components/Select/SelectContainer.js

```diff
@@ -109,11 +105,17 @@ const SelectContainer = forwardRef(
       // need to wait for Drop to be ready
       const timer = setTimeout(() => {
         const optionsNode = optionsRef.current;
+        const clearButton = clearRef.current;
         if (onSearch) {
           const searchInput = searchRef.current;
           if (searchInput && searchInput.focus) {
             setFocusWithoutScroll(searchInput);
           }
+        } else if (clear && clearButton && clearButton.focus) {
+          setFocusWithoutScroll(clearButton);
+          if (clear.position === 'bottom') {
+            setActiveIndex(0);
+          }
```

Comment:
I think we can remove this and let the code below handle setting active index below
```suggestion
```